nginx.conf

```diff
@@ -167,4 +167,56 @@ http {
             return 403;
         }
     }
+
+      map $uri:$status:$request_method $loggable_module {
+            ~(\.tar\.gz)\:200\:GET$ 1;
+            default 0;
+         }
+
+        map $uri $module {
+            ~/modules/(?<captured_module>[^/]*)(.*(\.tar\.gz)) $captured_module;
+        }
+
+        log_format json escape=json
+            '{'
+            '"time_local":"$time_local",'
+            '"module":"$module",'
+            '"upstream_addr":"$upstream_addr",'
+            '"request":"$request",'
+            '"body_bytes_sent":"$body_bytes_sent",'
+            '"request_time":"$request_time",'
+            '"http_user_agent":"$http_user_agent"'
+            '}';
+
+       server {
+            listen 81;
+            error_page 404  /404.html;
+            set $bucket "cfbs.s3.amazonaws.com";
```

Comment:
Having `server_name` here would be nice.

Reply:
added